frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
-  "homepage":"https://portfolio.sweetsofimc.dev/",
```

Comment:
This is actually a backup site, and it's deployed through a different pipeline, so modifying it would cause that separate pipeline to fail